dataactbroker/handlers/accountHandler.py

```diff
@@ -802,13 +810,13 @@ def set_skip_guide(self, session):
                 )
             skip_guide = request_dict['skip_guide']
             if isinstance(skip_guide, bool):    # e.g. from JSON
-                user.skip_guide = skip_guide
+                g.user.skip_guide = skip_guide
             elif isinstance(skip_guide, str):
                 # param is a string, allow "true" or "false"
                 if skip_guide.lower() == "true":
```

Comment:
Is it possible to simplify the assignment of `skip_guide` here? just curious if there's a way to handle these checks/conversions.

found some references here but nothing seems like a catch all for the situations here: http://stackoverflow.com/questions/715417/converting-from-a-string-to-boolean-in-python